tensorpipe/channel/context.h

```diff
@@ -97,6 +96,10 @@ class Context {
 
   virtual ~Context() = default;
 
+  // FIXME: This is a private, temporary, API. It will be removed in the
+  // coming weeks. DO NOT USE IT.
+  virtual bool supportsDeviceType(DeviceType type) const = 0;
```

Comment:
Here is the temporary method for determining which type of buffer a channel supports.